### PR TITLE
Updated for v0.2.0 release an minor fixes.

### PR DESCRIPTION
**Pull request by Jouaux**
Updated and fixed German Translation

### CODE REVIEW

File: po/de.po [hunk 1]
```diff
@@ -25,25 +25,86 @@ msgid "Extension Manager"
...
+#: data/com.mattjakeman.ExtensionManager.gschema.xml:11
+msgid "Colour Scheme Preference"
+msgstr "Farbschema Einstellungen"
+
+#: data/com.mattjakeman.ExtensionManager.gschema.xml:12
+msgid ""
+"Whether to follow the system colour scheme or force either light or dark "
+"mode."
+msgstr ""
+"Ob es wie das System-Farbschema oder Dunkler oder Heller Schema erzwungen "
```
Comment (badcel) [inline po/de.po:105]: This is no German sentence. What about:

Ob das System-Farbschema oder der helle oder dunkle Modus bevorzugt werden soll.

File: po/de.po [hunk 2]
```diff
@@ -54,63 +115,133 @@ msgstr ""
...
+
+#: src/exm-browse-page.blp:7
+msgid "Search for extensions"
+msgstr "Suchen Sie nach Erweiterungen"
+
+#: src/exm-browse-page.blp:8
+msgid ""
+"Enter a keyword to search 'extensions.gnome.org' for GNOME Shell Extensions."
+msgstr ""
+"Geben Sie ein Keyword ein, um auf 'extensions.gnome.org' nach GNOME Shell "
```
Comment (badcel) [inline po/de.po:128]: Keyword -> Suchbegriff?

File: po/de.po (hunk 1 above)
Comment (Jouaux) [reply po/de.po]: Oh overlooked that.

File: po/de.po (hunk 2 above)
Comment (Jouaux) [reply po/de.po]: That was the word is was looking for thanks.

**Comment by mjakeman** [comment]
Thanks for the updates! (and @badcel for review)

It all looks good to me, let's go ahead and merge.

We've just added a new `translator-credits` string in master, so translators appear in the about dialog. Could you (and also @antifallobst) open a new PR and add yourselves to that?

**Pull request merged**